Makefile

```diff
@@ -1,8 +1,10 @@
 .PHONY: build test
 
+VERSION=$(shell git describe --tags --dirty --always)
+
 build:
-	go build -o conduit-connector-log cmd/connector/main.go
+	go build -ldflags "-X 'github.com/conduitio/conduit-connector-file.version=${VERSION}'" -o conduit-connector-log cmd/connector/main.go
```

Comment:
Typo: conduit-connector-**file**

Reply:
HOW!? I changed this, I swear 😢

Reply:
Maybe you didn't push the latest commit?

Reply:
I changed it in `.goreleaser.yml` but not in the Makefile, my bad!